src/lib/registry.ts

```diff
@@ -203,7 +217,7 @@ export class DappStoreRegistry {
         debug("registry changed. updating...");
         this.cachedRegistry = remoteRegistry;
         this.lastRegistryCheckedAt = new Date();
-        this.searchEngine.addDocuments(this.cachedRegistry.dapps);
+        // this.searchEngine?.addAll(this.cachedRegistry.dapps);
```

Comment:
Index should also happen here

Reply:
if we add same document more than one time, than it is throwing an error. indexing already happening in buildSearchIndex function after calling registry.

Reply:
ok